src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+class TodoForm extends Component{
+    
+   
+
+    render() {
+        return(
+            <div>
+                <input onChange={this.props.valueChange}></input>
```

Comment:
And then on the input, add the value attribute like this => 
`<input onChange={this.props.valueChange} value={this.props.val} />`